src/Server/GraphiteFinder.cpp

```diff
@@ -0,0 +1,273 @@
+
+
+#include "GraphiteFinder.h"
+#include "GraphiteUtils.h"
+#include "fmt/format.h"
+#include <regex>
+#include <chrono>
+
+namespace DB
+{
+
+
+int IndexDirect = 1;
```

Comment:
Having global non-const variables in the DB namespace is probably not the best idea.